### PR TITLE
[IT-2594] Rename host for lambda-mips-api

### PR DESCRIPTION
**Pull request by jesusaurus**
Prepare for having two finops lambdas under the same domain by creating a zone and wildcard cert for `finops.sageit.org`, replacing the secondary wildcard certificate for `sageit.org`.

### CODE REVIEW

File: org-formation/100-shared-dns/_tasks.yaml [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+    Account: !Ref SageITAccount
+  Parameters:
+    DnsDomainName: "finops.sageit.org"
+
+# Wildcard certificate for '*.finops.sageit.org'
+SageITFinopsCertificate:
   Type: update-stacks
   Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/acm-certificate.yaml
-  StackName: !Sub '${resourcePrefix}-sageit-cert'
+  StackName: !Sub '${resourcePrefix}-sageit-finops-cert'
```
Comment (zaro0508) [inline org-formation/100-shared-dns/_tasks.yaml:20]: we need to be very careful here.  a rename will make org-formation delete an existing stack.  We don't want to delete the existing stack because there are dependencies on it.

The better process is to add new resources then carefully deprecate existing resources.

File: org-formation/100-shared-dns/README.md [hunk 2]
```diff
@@ -12,16 +12,12 @@ used by the hosting service.
 
 Each wildcard cert will need to be manually verified before it can be used.
 
-### sageit.org
+### finops.sageit.org
```
Comment (zaro0508) [inline org-formation/100-shared-dns/README.md:15]: What is the motivation behind this change?  there's nothing in the referenced jira issue that describes why your doing this.

We still need `sageit.org` because we have `vpn.sageit.org`, `sc.sageit.org`, etc..

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/100-shared-dns/README.md (hunk 2 above)
Comment (jesusaurus) [reply org-formation/100-shared-dns/README.md:15]: We will still have both a `sageit.org` zone and wildcard certificate in the sageit account, this is only removing the use of the `sageit.org` zone in the admincentral account by removing the wildcard certificate from that account.

The motivation is to have friendly names for both finops lambdas, I had the wrong ticket in the subject but I've updated it to IT-2594

File: org-formation/100-shared-dns/_tasks.yaml (hunk 1 above)
Comment (jesusaurus) [reply org-formation/100-shared-dns/_tasks.yaml:38]: fortunately the only dependency on this is the mips-api lambda, but since it's managed earlier than this stack (in `050-costs`) you're right we need to wait until after we've migrated the lambda to the new service before we can remove deprecated cert

Comment (zaro0508) [reply org-formation/100-shared-dns/_tasks.yaml:38]: as a separate matter, i'm wondering if would make sense to move setup of shared-dns to before setup of costs?

File: org-formation/100-shared-dns/_tasks.yaml [hunk 3]
```diff
@@ -18,7 +19,33 @@ SageItCertificate:
   Parameters:
     DnsDomainName: 'sageit.org'
 
-SagebioAppZone:  # Zone for application front-end DNS names, e.g. 'dca.app.sagebionetworks.org'
+# Zone for finops lambdas in admincentral
+SageITFinopsZone:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/R53-hostedzone.yaml
+  StackName: !Sub '${resourcePrefix}-sageit-finops-zone'
+  StackDescription: Create a shared hosted zone for application domains
```
Comment (zaro0508) [inline org-formation/100-shared-dns/_tasks.yaml:27]: could we be more specific with this description?

**Review by zaro0508 — APPROVED** [review]
generally LGTM, just need to update description and i had a question

File: org-formation/100-shared-dns/_tasks.yaml (hunk 1 above)
Comment (jesusaurus) [reply org-formation/100-shared-dns/_tasks.yaml:38]: yeah, I regret the ordering of a lambda in `050-costs` and the dns and certificates in `100-shared-dns`, but I'm not sure which would be better to move
